python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -35,6 +35,54 @@
 import lsst.meas.algorithms as measAlg
 import lsst.meas.algorithms.utils as maUtils
 import lsst.meas.extensions.psfex as psfex
+from lsst.pipe.base import AlgorithmError
+
+
+class PsfexTooFewStarsAlgorithmError(AlgorithmError):
+    """Custom exception for PSFEx algorithmic error when too few good stars are
+    available for PSF determination.
```

Comment:
Maybe something like this?

```suggestion
    """Raised if too few good stars are available for PSF determination.
```